Packs/GitHub/ReleaseNotes/1_3_10.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### GitHub
+- Fixed an issue where the **GitHub-get-branch** command failed to parse response in case there was no PR author.  
```

Comment:
no PR or no PR author?

Reply:
no PR author

Reply:
how is that possible? deleted user?

Reply:
Also might be. Tho I think in this case it's not related it is this branch https://github.com/demisto/content-private/pull/473